client/_library.sass

```diff
@@ -0,0 +1,15 @@
+
+// Colour Scheme	
+$main-colour: #D4E6E3 !default
```

Comment:
I've never seen this `!default` before. Neat

Reply:
Thank you Sass documentation. We'll see if it ends up being used, but I figured it would be good to include it at the start.